### ISSUE TITLE
include context with `let_it_be` in RSpec's config

### ISSUE THREAD

**Issue by WojciechKo** [palkan/test-prof#117]
### Tell us about your environment

**Ruby Version:2.5.3**

**Framework Version: RSpec 3.8**

**TestProf Version: 0.7.3**

### What did you do?
I want to create a context that'll be included in all of the specs that requires `rails_helper`

I've added following content to `rails_helper.rb`:
```ruby
require 'test_prof/recipes/rspec/let_it_be'
require 'test_prof/recipes/rspec/before_all'

RSpec.shared_context :regions do
  let_it_be(:global_region) { FactoryBot.create(:region, :global) }
  let_it_be(:poland_region) { FactoryBot.create(:region, :poland) }
end

RSpec.configure do |config|
  config.include_context :regions
end
```

### What did you expect to happen?
Have access to `global_region` and `poland_region` from each of the spec's that require `rails_helper`

### What actually happened?
```console
Failure/Error: let_it_be(:global_region) { FactoryBot.create(:region, :global) }

NoMethodError:
  undefined method `let_it_be' for RSpec::ExampleGroups::SerpCheckChannel:Class
```
I'm able to use created context within a single spec with `include_context :regions_spec` inside of the spec, but I'd rather include this context once for all of the specs.

**Comment by palkan** [comment]
Thanks for the report.

That's a known (though not published) issue, somehow related to the order of the modules inclusion in RSpec.

The plan is to fix this in the next release (~1 month). I'll try to provide a temp fix sooner, here, in the issue.

**Comment by palkan** [comment]
Fixed by https://github.com/palkan/test-prof/commit/4beaf4043057c761aee7034786e75cc3a6276237

Released as 0.7.5.

/cc @Earendil95
